bin/micro

```diff
@@ -12,25 +12,24 @@ import transformAsync from 'babel-plugin-transform-async-to-generator'
 import transformRuntime from 'babel-plugin-transform-runtime'
 import alias from 'babel-plugin-module-alias'
 
-program
-.version(version)
-.option('-p, --port <port>', 'Port to listen on (3000)', parseInt)
-.option('-n, --no-babel', 'Skip Babel transformation')
-.parse(process.argv)
+args
+  .option('port', 'Port to listen on', 3000, parseInt)
```

Comment:
@rauchg By default, args uses the first letter of the long version of the option. If you don't want that, you can provide an array instead: `['p', 'port']`